docs/blog/20240624-Daeraxa-3kStars.md

```diff
@@ -0,0 +1,136 @@
+
+<iframe width="560" height="315" src="https://www.youtube.com/watch?v=Y7aEiVwBAdk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Microsoft acquires GitHub
+
+<img src="./assets/microsoft-github.png" height="200" />
+
+In June of 2018, Microsoft announced that they were [acquiring GitHub](https://blogs.microsoft.com/blog/2018/06/04/microsoft-github-empowering-developers/). This immediately raised concerns from the Atom user base because Microsoft had released their own Electron-based text editor back in 2015, Visual Studio Code. The concerns were that Atom would shut down development of Atom in favor of their own product as the two products were competing in very similar areas. To help alleviate the Atom community's worries, Microsoft's newly instated Github CEO, Nat Friedman, said in a [Reddit AMA](https://www.reddit.com/r/AMA/comments/8pc8mf/im_nat_friedman_future_ceo_of_github_ama/e0a2b2e/) that Atom was safe and development would continue on both products.
+
+### Tree-sitter lands in Atom
+
+<img src="./assets/tree-sitter-lands.png" height="200" />
+
+You may have read our [series of blog posts](https://pulsar-edit.dev/tag/tree-sitter/) from [@savetheclocktower] and [@maurício szabo] that detail the huge amount of work that has been going on to upgrade our Tree-sitter implementation. This all goes back to 2018 when [Max Brunsfeld announced](https://github.blog/2018-10-31-atoms-new-parsing-system/) that Atom 1.32 would be shipping with a brand new parsing system that promised to be better at understanding your code and making it easier to write - Tree-sitter. Like how Atom Shell was created for Atom and was then widely used elsewhere, the same can be said of Tree-sitter, which now sits at the heart of a number of other projects: Helix, Zed, and even Github's own [Linguist](https://github.com/github-linguist/linguist) library.
+
+### GitHub sunsets Atom
+
+<img src="./assets/sunsetting2.png" height="200" />
+
+So, having just mentioned how Microsoft had promised that Atom and VS Code would both see active development and support, it became clear that interest in Atom development was decreasing, new features weren't forthcoming, and issues were being left unresolved. The Atom.io blog page saw its last update in 2019 with the announcement of Atom 1.39, and most releases after that featured fixes and updates, but little in the way of exciting new features. The old Atom forum was shut down (but it is still browsable via the [Wayback Machine](https://web.archive.org/web/20210622052601/https://discuss.atom.io/)), and moved to the, then new, [GitHub Discussions](https://github.com/atom/atom/discussions) page, but there was little interaction from the original developers. The original package website was becoming unmaintained and hadn't had the "Featured" or "Trending" packages updated in a long time.
```

Comment:
```suggestion
Despite the reassurances that Atom and VSCode could coexist under Microsoft’s aegis, it became clear that interest in Atom development was decreasing, new features weren't forthcoming and issues were being left unresolved. The Atom.io blog page saw its last update in 2019 with the announcement of Atom 1.39; most releases after that featured fixes and updates but little in the way of exciting new features.

The old Atom forum was shut down (but is still browsable via the [Wayback Machine](https://web.archive.org/web/20210622052601/https://discuss.atom.io/) and moved to the then-new [GitHub Discussions](https://github.com/atom/atom/discussions)… but there was little interaction from the original developers. The original package website languished and didn't have its "Featured" or "Trending" packages updated for a long time.
```

Reply:
Whilst I love the wording with "aegis", I tend to stay away from slightly more flowery language simply because I know we have a somewhat international audience. I feel this might get lost.

Reply:
You’re free to pick a simpler synonym. The main thing here was simplifying the transition.

Reply:
I've changed it to Umbrella, I honestly do prefer your one still but I think it wouldn't have hit the mark with many.